src/test/java/Exercise5Test.java

```diff
@@ -0,0 +1,14 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class Exercise5Test {
+    String s = new String("1 1 2 3 5 8");
+    String s1 = new String("1 1 2 3 5 8 13 21");
+    @Test
+    public void Test_Excercise5(){
+        assertEquals(new Exercise5().fibonaci(9), s);
```

Comment:
Không được khai báo biến như thế này
Thay vào đó

```suggestion
        assertEquals(new Exercise5().fibonaci(9), "1 1 2 3 5 8");
```

Reply:
Done